packages/webapp/src/util/convert-units/unit.js

```diff
@@ -253,6 +253,21 @@ export const seedYield = {
   databaseUnit: databaseUnit.mass,
 };
 
+export const pricePerSeedYield = {
+  metric: {
+    units: ['kg', 'mt'],
+    defaultUnit: 'kg',
+    breakpoints: [1000],
+  },
+  imperial: {
+    units: ['lb', 't'],
+    defaultUnit: 'lb',
+    breakpoints: [2000],
+  },
+  databaseUnit: databaseUnit.mass,
+  invertedUnit: true,
+};
```

Comment:
Maybe we can remove the breakpoints for now?

My concern:
1. User's unit system is "imperial", and user inputted "460$/lb" (= 1014$/kg)
2. The user changed unit system to "metric"
3. The value (460$/lb) is now displayed as "1014000$/mt" (it's okay if what the user wants to see is 1014000$/mt rather than 1014$/kg)